### DISCUSSION TITLE
Seeking Qualified Developers and Experts in Web3

### DISCUSSION THREAD

**Discussion by sl2023** [community/community#37349] [Other Feature Feedback, Questions, & Ideas]
Hello GitHub Community

We are currently seeking help from developers, for a concept that is aimed to integrate Web2 Desktop with Web3 but focuses its use case on a specific sector of owners, creators, authors, and publishers of digital media content. 

This concept known as "**SL**" has never been done before and so it would require creating a homegrown product that supports and _integrates Web2 Desktop_ (_Digital Audio Media Interface, an environment where owners, creators, authors, and publishers reside_) with _Web3 Blockchain and NFTs_.

My partner and I would like to elaborate more, once we can find the right skill set and talent via the GitHub community. 

Can anyone here provide any leads, as to where we may begin to have such a discussion with qualified developers and experts in Web3 and who can help bridge Web2 desktop applications with Web3?

Thanks in advance (Ced and Brian).

**Comment by ossanna16** [comment]
Hi @sl2023, we're happy you're here 😀
However, we currently do not allow job listings on the Community on Discussions. It is something that we've added to the list to investigate, if we can do so without overwhelming other conversations. Keep checking back as the Community grows and we add more capabilities!
